File: CiscoPCCE.Toolkit/Bean/LicenseReservationReturnCode.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlType("licenseReservation")]
    public class LicenseReservationReturnCode : BaseApiBean
    {
        [XmlElement("returnReservationCode")]
        public string? ReturnReservationCode { get; set; }
    }
}

File: JavaSrc/toolkit/bean/LicenseReservationReturnCode.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="licenseReservation")
public class LicenseReservationReturnCode extends BaseApiBean {
  private String returnReservationCode;

  public String getReturnReservationCode() {
     return this.returnReservationCode;
  }

  public void setReturnReservationCode(String returnReservationCode) {
     this.returnReservationCode = returnReservationCode;
  }


}
